DeepMenu/Border/MuBorderType.swift

```diff
@@ -14,6 +15,7 @@ enum MuBorderType {
         get {
             switch self {
                 case .pod   : return "z⃝"
+                case .slider : return "-"
```

Comment:
@musesum you may want a niftier symbol here, if you agree that having a 1D slider is worthwhile to support. Note that this is just a placeholder in any case, nothing is using it yet.